### DISCUSSION TITLE
Text in complex line types not being imported

### DISCUSSION THREAD

**Discussion by khcon** [mozman/ezdxf#809] [Issues]
Hello,

I believe I've found a bug concerning complex linetypes. I reached out on [stackoverflow](https://stackoverflow.com/questions/74789980/is-it-possible-to-fetch-custom-linetypes-using-ezdxf), but the solution that was provided had already been tested. I spent some time making absolutely sure that it wasn't my data that was causing this.

Following the instructions previously mentioned in the stackoverflow question, the linetypes keep their "simple" type (eg : dashed), but the text that belongs to the linetype disappears. In its place there seems to be a DFX point which would be where the text's anchor would be.

Here are a coupe of screenshots and a bare-bones DXF for reproduction :
 
Original file : 

![image](https://user-images.githubusercontent.com/121307228/209367743-70dfe5c6-f14a-44ce-a811-746c5937e55c.png)

After 

`import ezdxf`
`from ezdxf.addons import Importer`
`sdoc = ezdxf.readfile(r'(...)\complex_lines.dxf')`
`tdoc = ezdxf.new()`
`importer = Importer(sdoc, tdoc)`
`importer.import_tables()`
`importer.import_modelspace()`
`importer.import_paperspace_layouts()`
`importer.finalize()`
`tdoc.saveas(r'(...)\complex_lines_copy.dxf')`

![image](https://user-images.githubusercontent.com/121307228/209368374-37fc0671-84fd-40ab-8006-fa8b4f133d84.png)

[complex_lines.zip](https://github.com/mozman/ezdxf/files/10296162/complex_lines.zip)

**Comment by mozman** [comment]
Importing complex linetypes works, as BricsCAD shows:

![image](https://user-images.githubusercontent.com/4230966/209388665-f6ef905f-277f-4b7d-8a13-0677ef96c31f.png)

So some scaling options or anything else are not imported and I don't waste time searching for that (it is not the linetype scaling).

If you want to know why, check out the `Importer` add-on [docs](https://ezdxf.mozman.at/docs/addons/importer.html):

![image](https://user-images.githubusercontent.com/4230966/209388978-0aaacec1-ff6f-4d46-91f0-a22dafffa9bd.png)
